.babelrc

```diff
@@ -22,7 +22,7 @@
     },
     "test": {
       "presets": [
-        "@babel/react",
+        ["@babel/react", { "useBuiltIns": true }],
```

Comment:
this will break older browsers, personally wouldn't advise this

Reply:
@Andarist It's test only. Puppeteer cannot read from node scope where babel place imports, so this allow just to use `Object.assign` instead of helper.

Reply:
Oh, I haven't noticed it's only for tests 🙈 this is fine then!